docs/src/components/ColorSwatch/ColorSwatch.component.tsx

```diff
@@ -0,0 +1,110 @@
+// REACT
+import React, { useState } from 'react';
+// VENDOR
+import styled from 'styled-components';
+import { ColorHelpers } from '../ColorHelpers';
+import { Pill } from '../Pill'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { Typography } from '../../../../src/Typography';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchProps {
+  colorHexCode: string;
+  colorName: string;
+}
+
+
+// STYLES
+const StyledColorSwatch = styled.div`
+  border: 0.5rem solid ${props => props.highContrastBorderColor}};
+  display: inline-block;
+  margin: 0 2rem 2rem 0;
+  width: 17rem;
+
+  .color-swatch-pill-container {
+    background: ${props => props.colorHexCode };
+    cursor: pointer;
+    padding: 7rem 1rem 1rem 1.5rem;
+  }
+
+  .color-swatch-text-container {
+    border-top: 0.05rem solid ${props => props.highContrastBorderColor};
+    padding: 1rem;
+  }
+
+  .color-swatch-text-container p {
+    margin: 0;
+    padding: 0.25rem 0 0.25rem 0.5rem;
+  }
+
+  .color-swatch-text-container p:hover {
+    background: ${colors.ash.light};
+    border-radius: 1rem;
+    cursor: pointer;
+    text-decoration: underline;
```

Comment:
```
...
cursor: pointer;
text-decoration: underline;
...
```

If you're writing all of these styles that are defaults for the anchor tag, wouldn't it be better to use the anchor tag?

Reply:
This is a good question. In this case, I would rather add a few anchor-like qualities to `<p>` tags than use `<a>` and have to subtract other default styles I don't want (blue color, inline display, etc.).